### PR TITLE
How to remove a project

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
Add a page to detail how to remove a project from CircleCI

# Reasons
This content was missing

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/stop-building-a-project-on-circleci.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Learn how to remove a project from your CircleCI account
+:icons: font
+:toc: macro
+:toc-title:
+
+[#introduction]
+== Introduction
+This page provides details on how to stop builds for a project on CircleCI, and how to remove all projects data when required.
```
Comment (rosieyohannan) [inline jekyll/_cci2/stop-building-a-project-on-circleci.adoc:18]:
```suggestion
This page provides details on how to stop builds for a project on CircleCI, and how to remove all project data when required.
```

File: jekyll/_cci2/stop-building-a-project-on-circleci.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+:toc-title:
+
+[#introduction]
+== Introduction
+This page provides details on how to stop builds for a project on CircleCI, and how to remove all project data when required.
+
+[#stop-building-a-project]
+== Stop building a project
+
+CAUTION: The **Stop Building** option will not automatically remove all project data from the CircleCI databases. To remove all project data from CircleCI you need to contact CIrcleCI support, as detailed in the <<remove-a-project-and-all-associated-data>> section.
```
Comment (rosieyohannan) [inline jekyll/_cci2/stop-building-a-project-on-circleci.adoc:23]:
```suggestion
CAUTION: The **Stop Building** option will not automatically remove all project data from the CircleCI databases. To remove all project data from CircleCI you need to contact CircleCI support, as detailed in the <<remove-a-project-and-all-associated-data>> section.
```

**Review by kbatuigas — CHANGES_REQUESTED** [review]
Typo and a question

**Review by kbatuigas — APPROVED** [review]
🎉

**Pull request merged**